BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+load("//:gax_java.bzl", "google_java_format", "google_java_format_verification")
+
+exports_files(["dependencies.properties"])
+
+JAVA_COPTS = [
+    "-source",
+    "1.7",
+    "-target",
+    "1.7",
+]
+
+java_plugin(
+    name = "auto_value_plugin",
+    processor_class = "com.google.auto.value.processor.AutoValueProcessor",
+    deps = ["@com_google_auto_value_auto_value//jar"],
+    visibility = ["//visibility:public"],
+)
+
+# google-java-format
+java_binary(
+    name = "google_java_format_binary",
+    jvm_flags = ["-Xmx512m"],
```

Comment:
Should we add a comment about why this is needed? (I guess because otherwise jvm can crash because of not enough memory?)

Reply:
It is needed to run inside CircleCI (to avoid the build consuming more resources than actually allowed by CircleCI). I add the comment.